### PR TITLE
Add Glutton parachain to pr-custom-review

### PR DESCRIPTION
**Pull request by NachoPal**
(Added to the exclusion as the parachain has no economic value)

### CODE REVIEW

**Review by bkontur — APPROVED** [review]

**Review by gilescope — APPROVED** [review]

File: .github/pr-custom-review.yml
Comment (joepetrowski) [inline .github/pr-custom-review.yml:9]: > (Added to the exclusion as the parachain has no economic value)

This seems to do the opposite

Comment (NachoPal) [reply .github/pr-custom-review.yml:9]: Yeah, not my comment, sorry. I followed the same rules as the rest of the paras, but if it is not necessary because of "no economic value", then we just need to close this PR.

Comment (joepetrowski) [reply .github/pr-custom-review.yml:9]: The PR title and description contradict each other. The exclusion only applies to things that are getting two reviews from locks developers. I don't see the need to change it, I think 2 core devs are OK.

Comment (NachoPal) [reply .github/pr-custom-review.yml:9]: > The PR title and description contradict each other. 

Yeah, that happened because PR title and description comment are not from the same person :P

Agree, closing the PR.

**Pull request closed without merging**
